app/src/test/java/org/mozilla/vrbrowser/GleanMetricsServiceTest.kt

```diff
@@ -0,0 +1,26 @@
+package org.mozilla.vrbrowser
+
+import androidx.test.core.app.ApplicationProvider
+import mozilla.components.service.glean.testing.GleanTestRule
+import org.junit.Assert.assertEquals
+import org.junit.Rule
+import org.junit.Test
+import org.mozilla.vrbrowser.GleanMetrics.*
+
+
+@get:Rule
+val gleanRule = GleanTestRule(ApplicationProvider.getApplicationContext())
+
+class GleanMetricsServiceTest {
+
+    @Test
+    fun testMetricsGathering() {
+        assert(!Url.visits.testHasValue())
+        Url.visits.add()
```

Comment:
@Dexterp37. Can you help me take a look at this line? It shows
```
org.mozilla.vrbrowser.GleanMetricsServiceTest > testMetricsGathering FAILED
    java.lang.AssertionError at GleanMetricsServiceTest.kt:18

2 tests completed, 1 failed
```
when I run my unit test.

It looks like GleanTestService is not activated yet? `Url.visits` should not have values at the beginning.

Reply:
Yes, this seems to be the case. See my other comments.